src/reading/fromZip.js

```diff
@@ -16,6 +16,11 @@ export default ({ fileFilter, parser=gdalParser }) => {
   unzip(inStream, { fileFilter })
     .then(({ files, done }) => {
       finished(out, done)
+      if (files.length < 1) {
```

Comment:
length === 0 is better as the files can't be -1

Reply:
:+1: